### DISCUSSION TITLE
Multiple publish output files

### DISCUSSION THREAD

**Discussion by wel21** [microsoft/TemplateStudio#4632] [Q&A] [answered]
Hi, I'm new to this and just started using TemplateStudio. Can anyone help me with my issue? every time I install the nuget package Microsoft.EntityFrameworkCore.SqlServer, below error will occur

Found multiple publish output files with the same relative path: D:\Users\<user>\Projects\RL.POSINV\RL.POSINV\RL.POSINV\obj\x64\Debug\net7.0-windows10.0.19041.0\win10-x64\MsixContent\Microsoft.Web.WebView2.Core.dll, C:\Users\<user>\.nuget\packages\microsoft.web.webview2\1.0.864.35\lib\netcoreapp3.0\Microsoft.Web.WebView2.Core.dll

I already tried the suggestions from below but it didn't work.
https://stackoverflow.com/questions/69919664/publish-error-found-multiple-publish-output-files-with-the-same-relative-path
https://github.com/dotnet/sdk/issues/22716

ps: I don't have a direct reference to Microsoft.Web.WebView2.Core.dll

these are the projects and references
![image](https://user-images.githubusercontent.com/32331198/221386889-1da9e6fb-c1ab-46af-a555-c07ba912f411.png)

Either main project or the core project, referencing Microsoft.EntityFrameworkCore.SqlServer gives me the same error.

**Comment by mikebattista** [comment, marked as answer]
I believe you're running into https://github.com/microsoft/TemplateStudio/issues/4633.

**Reply by wel21** [reply]
It is indeed. What worked on me is to install the lower version of Microsoft.EntityFrameworkCore.SqlServer that doesn't have Azure in its dependency.

**Reply by mikebattista** [reply]
Great. Glad you're unblocked.

**Comment by robinwilson16** [comment]
This remains an issue with .NET 8 Net MAUI projects.
Is there a fix for this other than using an older version of Entity Framework which I think won't be compatible with 8?
Add a MAUI web view and SQL Server and your project is not able to be published.

**Reply by robinwilson16** [reply]
Actually after trying the other methods to exclude one file ignoring the duplicates does work although fees "untidy" to have to do this:

```
<PropertyGroup>
	<ErrorOnDuplicatePublishOutputFiles>false</ErrorOnDuplicatePublishOutputFiles>
</PropertyGroup>
```
